build.gradle

```diff
@@ -33,9 +33,20 @@ allprojects {
   apply plugin: 'spinnaker.project'
   apply plugin: 'groovy'
 
+  ext {
+    spinnakerDependenciesVersion = "0.47.0"
+  }
+
+  if (spinnakerDependenciesVersion.indexOf('-dev') != -1) {
```

Comment:
convenience for working with locally modified spinnaker-dependencies while updating things, intend to put this same change in other repos as I update them